include/fizzy/fizzy.h

```diff
@@ -273,8 +277,9 @@ void fizzy_free_instance(FizzyInstance* instance);
 
 /// Get pointer to module of an instance.
 ///
-/// @note  The returned pointer represents non-owning, "view"-access to the module and must not be
-///        passed to fizzy_free_module.
+/// @return  Pointer to a module.
```

Comment:
Well, even if it is basic, shouldn't the params also be included?

Reply:
Added.